File: Calculator/calculator_one.cpp
/*
 * Author: Christopher Forte
 * Date: October 19, 2022
 * Title: Calculator (C++)
 * Description: First attempt at a basic calculator in C++. 
 */

#include <iostream>
using namespace std;

int main()
{

  int num1, num2, total;
  char op;

  cout << "Enter First Operand: ";
  cin >> num1;
  
  cout << "Enter Operator: ";
  cin >> op;
  
  cout << "Enter Second Operand: ";
  cin >> num2;

  if (op == '+')
  {
    total = num1 + num2;
    cout << total;
  }

  if (op == '-')
  {
    total = num1 - num2;
    cout << total;
  }

  if (op == '*')
  {
    total = num1 * num2;
    cout << total;
  }

  if (op == '/')
  {
    total = num1 / num2;
    cout << total;
  }

  if (op == '%')
  {
    total = num1 % num2;
    cout << total;
  }

  return 0;
}

File: Calculator/calculator_one.py

#Author: Christopher Forte
#Date: October 21, 2022
#Title: Calculator (Python)
#Description: First attempt at a basic calculator in Python.


num1 = int(input("Enter First Operand: "))
ch = input("Enter Operator: ")
num2 = int(input("Enter Second Operand: "))

output = 0 

if ch == '+':
  output = num1 + num2

elif ch == '-':
  output = num1 - num2

elif ch == '*':
  output = num1 * num2

elif ch == '/':
  output = num1 / num2

elif ch == '%':
  output = num1 % num2

else:
  print("Invalid Input")

print(output)
